conans/model/info.py

```diff
@@ -170,12 +178,12 @@ def deserialize(data):
 class ConanInfo(object):
 
     def copy(self):
```

Comment:
``ConanInfo.create()`` requires more information, like "indirect_reqs", that it is not available from ConanInfo. I'd say to keep the use case and the model as simple as possible, just the bare minimum, and let's see its usage. I'd say that with ``self.settings``, ``self.options`` and the current ``self.info_build`` is more than enough for all possible cases I can think of.